governator-annotations/src/main/java/com/netflix/governator/annotations/AutoBindSingleton.java

```diff
@@ -22,10 +22,12 @@
 
 /**
  * Marks a class as a singleton. Governator will auto-bind it as an eager singleton
+ * @deprecated  AutoBindSingleton is deprecated in favor of bindings in a Guice Module.
```

Comment:
Same as above, a wiki might be nice that walks through the various uses and how to model them in the new way.